common/lib/xmodule/xmodule/js/src/problem/edit.js

```diff
@@ -573,6 +573,17 @@
                             );
                         },
 
+                        checkIsNumeric = function(stringValue) {
+                            // remove OLX feedback
+                            if ((stringValue.indexOf('{{') !== -1) && (stringValue.indexOf('}}') !== -1)) {
+                                stringValue = stringValue.replace(/{{[\s\S]*?}}/g, '').trim();
```

Comment:
Can you add some comment on what this replace is meant to do? Is it trying to remove all white space from this string?